web/packages/teleport/src/Integrations/Enroll/AwsConsole/Access/ProfilesTable.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+
+import Table, { Cell, LabelCell } from 'design/DataTable';
+import { ClientSidePager } from 'design/DataTable/Pager';
+import { StyledTable } from 'design/DataTable/StyledTable';
+import { FetchingConfig, PagedTableProps } from 'design/DataTable/types';
+import Flex from 'design/Flex';
+import * as Icons from 'design/Icon';
+import { Check } from 'design/Icon';
+import { P2 } from 'design/Text';
+
+import { RolesAnywhereProfile } from 'teleport/services/integrations';
+
+export function ProfilesTable({
+  data,
+  fetchStatus,
+  onFetchNext,
+  onFetchPrev,
+}: {
+  data: RolesAnywhereProfile[];
+} & Omit<FetchingConfig, 'onFetchMore'>) {
+  return (
+    <Table
+      data={data}
+      columns={[
+        {
+          altKey: 'selected',
+          headerText: '',
+          render: row =>
+            data?.includes(row) ? (
+              <Cell width="50px">
+                <Check size="small" color="success.main" />
+              </Cell>
+            ) : (
+              <Cell width="50px" />
+            ),
+        },
+        {
+          key: 'name',
+          headerText: 'Profile Name',
+        },
+        {
+          key: 'tags',
+          headerText: 'Tags',
+          render: row => <LabelCell data={row.tags} />,
+        },
+        {
+          key: 'roles',
+          headerText: 'IAM Roles',
+        },
```

Comment:
lets use the custom render, so that we can put a space between the roles (make it more readable)

<img width="85" height="177" alt="image" src="https://github.com/user-attachments/assets/d2d84c8d-6ac9-4758-b8b8-f6a7381a1666" />